.travis/install.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
```

Comment:
It won't hurt in `after_script` but to @jasp00's point, the `ccache` statistics aren't really critical for a pass/fail and it's a single-line, so `set -e` on the whole script hasn't yet been warranted over something like `exit $?`.  But @zapashcanon you're right, it's best to add it.